docs/designers-developers/developers/slotfills/plugin-document-setting-panel.md

```diff
@@ -0,0 +1,26 @@
+# PluginDocumentSettingPanel
+
+This SlotFill allows adding items below the Status & Availability panel in the Document Sidebar.
```

Comment:
I think the documentation should be focused more on the "semantics" and less on the UI (sidebar) because we won't guarantee that this panel will always be shown in that exact place. 

I think something like

```
This SlotFill allows registering a UI to edit Document settings
```

or something like that would work a bit better.

Reply:
Updated